### DISCUSSION TITLE
Added AUR package

### DISCUSSION THREAD

**Discussion by vestingz** [pimalaya/himalaya#51] [Show and tell]
Just discovered this, liked it and quickly made an AUR package [here](https://aur.archlinux.org/packages/himalaya-git/), in case you want to add this to the README :)

**Comment by soywod** [comment]
Thanks a lot dude :) I update the wiki right now!

**[EDIT]** here it is https://github.com/soywod/himalaya/wiki/Installation#from-aur

**Reply by vestingz** [reply]
No worries :) Thank you for this nice software!

**Comment by remche** [comment]
@vestingz the package is broken since tests wont run if there is no imap server running on localhost.

**Reply by soywod** [reply]
Yes indeed, integration tests should be discarded: `cargo test --lib`.

**Reply by vestingz** [reply]
Thanks for the heads up! :) I just pushed the fix to the AUR.

**Reply by soywod** [reply]
Thanks @vestingz!
